db/migrate/db_migrate_20200513201952_rename_source_type_ddf_strings_v2_rb.rb

```diff
@@ -0,0 +1,60 @@
+class RenameSourceTypeDdfStringsV2 < ActiveRecord::Migration[5.2]
```

Comment:
I'd rename this file to `20200513201952_rename_source_type_ddf_strings_v2.rb` that way it follows the rail standard

Reply:
uppps, sorry! :facepalm: